packages/components/src/utils/hooks/useExternalLink.ts

```diff
@@ -25,9 +25,13 @@ export function useExternalLink(): (
       if (analytics?.onCancel) analytics.onCancel()
     }
 
-    const onOKPress = () => {
-      if (analytics?.onConfirm) analytics.onConfirm()
-      return Linking.openURL(url)
+    const onOKPress = async () => {
```

Comment:
With these having been made async anyway to ensure the promise rejections are handled, not sure if we should also just do `if (canOpenURL)` first.

As noted, in storybook I observed no difference in behavior between canOpenURL and openURL in terms of getting warnings. Going into RN code, it mostly looks to just passthrough to native logic and canOpenURL and openURL at the JS level are identical.

Reply:
If the RN code and warnings look identical, I think the try/catch  should be sufficient.